### PR TITLE
Issue#158

### PR DESCRIPTION
**Pull request by obormot**
Sample output:
```python
IP6(v=6, fc=224, flow=0, plen=68, nxt=50, hlim=1, src='...', dst='...', 
extension_hdrs={50: IP6ESPHeader(spi=256, seq=68, length=68, data='...')})
```

### CODE REVIEW

File: dpkt/ip6.py [hunk 1]
```diff
@@ -370,7 +389,7 @@ def test_ip6_extension_headers():
     ip.extension_hdrs[51] = IP6AHHeader(ah)
     do = ';\x02\x01\x02\x00\x00\xc9\x10\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
     ip.extension_hdrs[60] = IP6DstOptsHeader(do)
-    assert (len([k for k in ip.extension_hdrs if (not ip.extension_hdrs[k] is None)]) == 5)
+    assert (len([k for k in ip.extension_hdrs if (ip.extension_hdrs[k] is not None)]) == 5)
```
Comment (kbandla) [inline dpkt/ip6.py]: > assert (len([k for k in ip.extension_hdrs if ip.extension_hdrs.get(k)])])

**Comment by kbandla** [comment]
Looks good. Especially the descriptive variable names :)

File: dpkt/ip6.py (hunk 1 above)
Comment (obormot) [inline dpkt/ip6.py]: @kbandla code golf?  :-)
```
assert len(ip.extension_hdrs)==5
```
I'll get it in with a next round of changes..

Comment (kbandla) [inline dpkt/ip6.py]: :grin: .. actually dont worry about this. its nothing

**Pull request merged**